src/Webservices/ConfigurationHelper.php

```diff
@@ -214,30 +215,30 @@ public static function getWebserviceConfig($client, $webserviceName, $version =
 	 */
 	public static function getWebserviceHelper($client, $webserviceName, $version = '', $path = '')
 	{
-		if (!empty($webserviceName))
+		if (empty($webserviceName))
 		{
-			$version = !empty($version) ? $version : '1.0.0';
-			$webservicePath = !empty($path) ? WebserviceHelper::getWebservicesPath() . '/' . $path : WebserviceHelper::getWebservicesPath();
+			return null;
```

Comment:
I guess technically per the docblock it should be an empty string rather than null :P